docs/jade-getting-started.md

```diff
@@ -312,49 +246,65 @@ psql --list
+* Hit create!
+
+## 12. Setup Environment Variable
+There are several ways to go about this, but here is one way that works. You can set up a Z-shell
+configuration to keep your system environment variables. If you don't already have one created, you
+can create one by running `touch ~/.zshrc`.
+Below you'll find a list of environment variables needed to run TDR and tests locally. When you run
+`./render-configs.sh`, it populates key and txt files with secrets from vault and environment-specific
+values. On daily setup, you'll always need to run the following two commands in order:
+```
+./render-configs.sh
+source ~/.zhsrc
+```
+An alternate is to run `./render-configs.sh -i` which will put the variables into your clipboard. You
+can then paste these values into an intellij bootRun or test run profile.
 
-## 12. Repository Setup
-
-### 1. Build `jade-data-repo`
-
-Follow the [Build and Run Locally](https://github.com/DataBiosphere/jade-data-repo#build-and-run-locally)
-section in the [main readme](https://github.com/DataBiosphere/jade-data-repo#jade-data-repository---)
-to build `jade-data-repo`.
+### Environment Variables
 
-* You will need to run `render-configs.sh` before running integration tests.
+While not exhaustive, here's a list that notes the important environment variables to set when running
+`jade-data-repo` locally. These variables override settings in jade-data-repo/application.properties.
+You can convert any application.property to an environment variable by switching to upper case and
+every "." to "_".
 
-* **Set Environment Variables**: While not exhaustive,
-  here's a list that notes the important environment variables to set when running `jade-data-repo` locally.
-  Instances of `ZZ` should be replaced by your initials or the environment (i.e. `dev`).
-  These variables override settings in jade-data-repo/application.properties.
-  You can convert any application.property to an environment variable by switching to upper case and every "." to "_".
+* Instances of `ZZ` are only needed if you have a personal development environment setup. It is no longer
+  recommended to set this up. But, if used, `ZZ` should be replaced by your initials or the environment (i.e. `dev`).
 
 ```
-# Point to your personal dev project/deployment
-export PROXY_URL=https://jade-ZZ.datarepo-dev.broadinstitute.org
 export JADE_USER_EMAIL=<EMAIL_YOU_CREATED_FOR_DEVELOPMENT>
 
-# Integration test setting: change this to http://localhost:8080/ to run against a local instance
-export IT_JADE_API_URL=https://jade-ZZ.datarepo-dev.broadinstitute.org
+# Integration test setting
+export IT_JADE_API_URL=http://localhost:8080
 
 # This file will be populated when you run ./render-configs.sh
 export GOOGLE_APPLICATION_CREDENTIALS=/tmp/jade-dev-account.json
 export GOOGLE_SA_CERT=/tmp/jade-dev-account.pem
 
-# Clears database on startup, test run, etc. This is further explained in the oncall playbook.
-export DB_MIGRATE_DROPALLONSTART=true
```

Comment:
This causes trouble for our embedded datasets in unit tests. It has been our workflow to manually run `./gradlew dropall` in order to clear out our database instead.

Reply:
It actually appears to be an issue with the `TERRA_COMMON_STAIRWAY_FORCECLEANSTART` variable getting set to true. (It need to be set to false for connected tests to pass). 

But, I still don't know that I recommend the drop all on start variable. We're now seeding our local environments with datasets that we don't want to automatically get deleted.